### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by iliakur** [twisted/towncrier#547] [Q&A] [answered]
When do you think the next release will drop? The trunk branch contains a fix for our usecase it would be nice to pull it in from a released version. Thanks in advance!

**Comment by iliakur** [comment]
**Update** turns out trunk **doesn't yet** have the fix that we need. I'm working on a PR.  To update the question:
Once this upcoming PR lands, when do we think we can cut a release?

**Reply by iliakur** [reply]
[PR up](https://github.com/twisted/towncrier/pull/548)

**Comment by adiroiban** [comment, marked as answer]
once the PR is merged, we can trigger a new release.

A new release is quite easy to make.
We just need a PR with the updated version and final release notes.
And once a tag is created, GitHub action will automatically build the package and publish on pypi
